modules/date-localizer/test/spec.coffee

```diff
@@ -17,6 +17,7 @@ describe 'dateTimeLocalizer', ->
   beforeEach ->
     hx.consoleWarning = chai.spy()
     localizer = hx.dateTimeLocalizer()
+    hx.preferences.timezone(hx._.preferences.defaultTimezoneLookup(0))
```

Comment:
The timezone changes in Chrome but not PhantomJS so we set it explicitly at the start of this test to prevent 'off by one' errors or problems when running in a different timezone